modules/ROOT/pages/runtime-manager-agent.adoc

```diff
@@ -23,47 +23,47 @@ The Runtime Manager is available only for Enterprise versions of Mule runtime en
 
 
 [NOTE]
-If you're deploying your applications to xref:deploying-to-pcf.adoc[Pivotal Cloud Foundry], then you don't need to worry about installing or configuring the Runtime Manager Agent. In that scenario, the Runtime Manager Agent functions as usual as the link between the Mule runtimes and the Runtime Manager. However, the same PCF buildPack that automatically creates new runtime instances also creates Runtime Manager Agent instances and registers these to the Runtime Manager. If you want to change the configuration of your Runtime Manager Agent instances, you must modify the buildPack.
+If you're deploying your applications to xref:deploying-to-pcf.adoc[Pivotal Cloud Foundry], then you don't need to worry about installing or configuring the Runtime Manager Agent. In that scenario, the Runtime Manager Agent functions as usual as the link between the Mule runtime engines and the Runtime Manager. However, the same PCF buildpack that automatically creates new runtime instances also creates Runtime Manager Agent instances and registers these to the Runtime Manager. If you want to change the configuration of your Runtime Manager Agent instances, you must modify the buildpack.
```

Comment:
in that scenario or in this scenario?

Reply:
I'd switch the 3rd and 4th sentences so the goal comes before all that info that the customer may not even care about unless they plan to configure their agent.